### DISCUSSION TITLE
How to avoid linting issue?

### DISCUSSION THREAD

**Discussion by PhilipZi** [neuefische/questions#147] [Web] [answered]
Hallo liebe Fische,

Ich würde gerne wissen wie ich diese Meldung im Bild auf neutral schalten kann, oder dass sich eslint damit nicht ausseinandersetzt?  
Gibt es da eine Möglichkeit? 

Bin super dankbar für jede hilfe! 

![Screenshot 2022-09-14 at 16 34 47](https://user-images.githubusercontent.com/102238180/190184798-4e29ea7a-0fa2-44c0-b89f-7167355b52b8.png)

**Comment by PhilipZi** [comment, marked as answer]
Hab es selber Lösen können, wenn man dort auf Quickfixes klickt kann man dort das ignorieren der gesammten Datei bzgl. dieses problems einstellen.

**Reply by pixelass** [reply]
Oder man folgt einfach der Empfehlung und verwendet `next/image`.
Next.js bietet eine Lösung an, die Bilder optimiert. Man kann dadurch Ladezeiten und Bildgrössen für verschiedene Displaygrößen optimieren und einige andere tolle features verwenden.

Siehe https://nextjs.org/docs/api-reference/next/image

Das ausschalten von Lintern ist generell nicht empfohlen.
